include/fmt/chrono.h

```diff
@@ -769,18 +769,24 @@ template <typename Char, typename Rep, typename OutputIt,
           FMT_ENABLE_IF(std::is_integral<Rep>::value)>
 OutputIt format_duration_value(OutputIt out, Rep val, int) {
   static FMT_CONSTEXPR_DECL const Char format[] = {'{', '}', 0};
-  return format_to(out, compile_string_to_view(format), val);
+
+  return vformat_to(out, to_string_view(format),
```

Comment:
Here and below: why not wrap `format` in `FMT_STRING` and use `format_to`?

Reply:
Because then it wouldn't compile on windows because of the MSVC error?

Reply:
AFAICS the MSVC bug is specific to static constexpr which we don't need here. I suggest replacing with:

```c++
  return format_to(out, FMT_STRING("{}"), val);
```

which seems to work: https://godbolt.org/z/Teqnhq.

Reply:
I thought we needed the static constexpr array so that the format string type would match the type of `Char`?

Reply:
On a second thought, we don't need `format_to` at all. This should be just `write<Char>(out, val)`.

Reply:
Excellent! I had no idea about that API. I've redone the other calls that were using format_to in chrono as well.